client/app/bundles/KudosApp/components/Kudo.jsx

```diff
@@ -96,67 +90,51 @@ export default class Kudo extends React.Component {
 
     return (
       <i
-        className={ likedBySelf ? "fas fa-heart" : "far fa-heart" }
-        onClick={ likedBySelf ? unlikeKudo(id) : likeKudo(id) }
+        className={likedBySelf ? 'fas fa-heart' : 'far fa-heart'}
+        onClick={likedBySelf ? unlikeKudo(id) : likeKudo(id)}
       />
     )
   }
 
   renderEditOptions() {
-    const Edit = () => (
-      <i
-        className="far fa-edit"
-        onClick={ this.makeEditable }
-      />
-    )
+    const Edit = () => <i className="far fa-edit" onClick={this.makeEditable} />
 
-    const Save = () => (
-      <i
-        className="far fa-save"
-        onClick={ this.update }
-      />
-    )
+    const Save = () => <i className="far fa-save" onClick={this.update} />
 
-    const Delete = () => (
-      <i className="far fa-trash-alt" onClick={ ()=> alert("delete")}></i>
-    )
+    const Delete = () => <i className="far fa-trash-alt" onClick={() => alert('delete')} />
 
     // TODO: enable Delete button when implemented in backend
-    return (
-      this.postedByActiveUser() ? (
-        <div>
-          { this.state.editing ? <Save /> : <Edit /> }
-          {/* <Delete /> */}
-        </div>
-      ) : null
-    )
+    return this.postedByActiveUser() ? (
+      <div>
+        {this.state.editing ? <Save /> : <Edit />}
+        {/* <Delete /> */}
```

Comment:
I don't think we're planning on supporting delete, so let's just remove this and the TODO.